File: source/phpfiles/weha/diff/SentenceDel.php
<?php
include_once('SentenceEdit.php');

class SentenceDel extends SentenceEdit {

	public function __construct($oldPos, $oldS, $mr) {
		$this->oldPos = $oldPos;
		$this->newPos = -1;
		$this->oldSentence = $oldS;
		$this->newSentence = null;
		$this->matchingRate = $mr;
	}
	
	public function getStartPos() {
		return $this->oldSentence->startPos;
	}
	
	public function getEndPos() {
		return $this->oldSentence->endPos;
	}
	
	public function getLength() {
		return $this->oldSentence->length;
	}
	
	public function descString() {
		return ("SentenceDel " . $this->oldSentence);
	}
}

File: weha/source/weha/wikianalysis/diff/sentence/SentenceDel.java
package mo.umac.wikianalysis.diff.sentence;

public class SentenceDel extends SentenceEdit {


	public SentenceDel(int oldPos, Sentence oldS, double mr) {
		super();
		this.oldPos = oldPos;
		this.newPos = -1;
		this.oldSentence = oldS;
		this.newSentence = null;
		this.matchingRate = mr;
	}
	
	public int getStartPos() {
		return this.oldSentence.startPos;
	}
	
	public int getEndPos() {
		return this.oldSentence.endPos;
	}
	
	public int getLength() {
		return this.oldSentence.length;
	}
	
	@Override
	public String descString() {
		return "SentenceDel(" + this.oldPos + ")";
	}
}
